Main.hs

```diff
@@ -1,25 +1,59 @@
+{-# OPTIONS_GHC -Wno-overlapping-patterns #-}
 module Main where
 
 import Control.Monad (unless)
 import Text.Printf (printf)
 
 short :: [a] -> Bool
-short = undefined 
+short [] = True
+short [x] = True
+short [x, y] = True
+short xs = False
 
```

Comment:
There is a way to implement this function in 2 lines. Can you do it? 

Also, avoid introducing variables names you aren't using. Instead, use wildcards.